lib/prompts/checkbox.js

```diff
@@ -85,7 +87,7 @@ Prompt.prototype.render = function (error) {
   var bottomContent = '';
 
   if (!this.spaceKeyPressed) {
-    message += '(Press <space> to select)';
+    message += `(Press ${chalk.cyan.bold('<space>')} to select, ${chalk.cyan.bold('<a>')} to toggle all, ${chalk.cyan.bold('<i>')} to inverse selection)`;
```

Comment:
Hey, this module is used all over the place. Let's keep it compatible with es5 otherwise it'll break for users on older node versions.